bundle/src/main/java/com/adobe/acs/commons/mcp/impl/ControlledProcessManagerImpl.java

```diff
@@ -133,7 +133,15 @@ public ProcessDefinition findDefinitionByNameOrPath(String nameOrPath) throws Re
     }
 
     private ProcessDefinition findDefinitionByName(String name) throws ReflectiveOperationException {
-        Class defClass = Class.forName(name);
+        Class clazz;
+        try {
+            clazz = Class.forName(name);
```

Comment:
I recognize that `Class.forName()` was already used here, so perhaps this should be a separate issue, but... is there a reason to use this vs. just looking up the processDefinition list? IIOW, should the code in the `catch` block be the *only* code path?

The problem is that `Class.forName()` won't work if the class is outside of the bundle. We can deal with that using `Dynamic-Package-Import`, but that's a pretty heavy weight solution.